app/models/forms.scala

```diff
@@ -13,13 +14,8 @@ object Forms {
 
   object TalkFormData {
     def apply(talk: Talk): TalkFormData = {
-      TalkFormData(
-        title = talk.title,
-        url = talk.url,
-        authors = talk.authors.foldLeft(Seq.empty: Seq[AuthorFormData]) { (seq, author) => seq :+ AuthorFormData(author) },
-        location = talk.location,
-        date = talk.date,
-        thumbnail = talk.thumbnail
+      transform[Talk, TalkFormData](talk,
+        "authors" -> talk.authors.foldLeft(Seq.empty: Seq[AuthorFormData]) { (seq, author) => seq :+ AuthorFormData(author) }
```

Comment:
I thought you could replace this fold with `talk.authors.map(author => AuthorFormData(author))`, but apparently you need `(talk.authors.map(author => AuthorFormData(author)): Seq[AuthorFormData])`. Not sure why the type inference breaks down here, but I think it's still marginally clearer.

Perhaps @cb372 might know what's going on.